source/class/qx/ui/core/scroll/MRoll.js

```diff
@@ -58,11 +58,6 @@ qx.Mixin.define("qx.ui.core.scroll.MRoll", {
      * @param e {qx.event.type.Roll} Roll event
      */
     _onRoll(e) {
-      // only wheel and touch
-      if (e.getPointerType() == "mouse") {
```

Comment:
Do we understand the intent of this original code? Is it really safe to just delete it? If we're not sure of the original intent, I'd be more comfortable adding an additional pointer type that's allowed, rather than allowing anything at all.  I think you'd said, in some conversation, that you were seeing a pointer type of "pen"? I'm not familiar with that pointer type, but if so, and it fixes the problem, then how about something like this?

```javascript
if ( [ "mouse", "pen" ].includes(e.getPointerType())) {
```

Reply:
I would like to know why the original code allows all type events except mouse. Judging by comment which is there right now code should work only for wheel and touch but nothing about pen (stylus). Probably pen and touch events are the same (pen like I think its just very small area of touch). As I said in the issue pen event generated in debian chrome (and chromium too) and I could drag inner content of container.Scroll. This mixin influences only several widgets: container.Scroll and List. I checked behaviour of list in other gui widget frameworks (Windows Forms .NET) and there you cant scroll content of a list by dragging. Maybe it is only reason. But for me it is nothing bad to scroll list by dragging (any disadvantages?).

Reply:
@derrell your code forbidden another event, on the contrary I want allow mouse (pen already allowed).

Reply:
Yes, sorry about that. I really wonder why mouse was excluded in the first place...